extensions/wikia/ArticleVideo/ArticleVideo.hooks.php

```diff
@@ -41,24 +41,26 @@ public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 		return true;
 	}
 
-	public static function onMakeGlobalVariablesScript( array &$vars, OutputPage $out ) {
+	public static function onMakeGlobalVariablesScript( array &$vars/*, OutputPage $out*/ ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Don't use an OnMakeGlobalVariablesScript hook handler to register JS variables. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AOnMakeGlobalVariablesScript)